### PR TITLE
Task/iguala atomic service

### PR DESCRIPTION
**Pull request by juliannieb**
#### What's this PR do?

Create an atomic web service for the creation of an iguala and its art types and quantities.
It also includes api-auth to the urls to login directly in the django REST service.

#### Where should the reviewer start?

- works/views.py
- works/serializers.py
- works/test_views.py

#### How should this be manually tested?

- flake8
- python manage.py test
- python manage.py runserver

#### Any background context you want to provide?

This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)

### CODE REVIEW

File: balarco/urls.py [hunk 1]
```diff
@@ -21,4 +21,5 @@
     url(r'^api/clients/', include('clients.urls')),
     url(r'^api/users/', include('users.urls', namespace='users')),
     url(r'^api/works/', include('works.urls', namespace='works')),
+    url(r'^api-auth/', include('rest_framework.urls', namespace='rest_framework')),
```
Comment (marccolm) [inline balarco/urls.py]: Wasn't this url supposed to be removed?

File: works/serializers.py [hunk 2]
```diff
@@ -19,20 +19,24 @@ class Meta:
         fields = ('id', 'work_type', 'name',)
 
 
-class IgualaSerializer(serializers.ModelSerializer):
+class ArtIgualaSerializer(serializers.ModelSerializer):
```
Comment (marccolm) [inline works/serializers.py]: I don't completely agree with the name 'Iguala', but ok I guess

File: balarco/urls.py (hunk 1 above)
Comment (juliannieb) [inline balarco/urls.py]: I realized why I had it, it's for login functionality in the Django REST framework to manually test the Web Services

File: works/serializers.py (hunk 2 above)
Comment (juliannieb) [inline works/serializers.py]: Jajajaja :C sorry

**Pull request merged**